### DISCUSSION TITLE
[Question]: When does retention deleting run?

### DISCUSSION THREAD

**Discussion by jamesonuk** [blakeblackshear/frigate#12306] [Ask A Question] [answered]
### What is your question:

Just trying to establish when the deletion of old recordings based on retention settings actually takes place.   And whether there is any way to control this?

**Comment by NickM-27** [comment, marked as answer]
by default it runs every hour starting when frigate starts. The interval is controllable

**Reply by jamesonuk** [reply]
Where is the config to control the interval?   I can't seem to spot anything in the config

**Reply by NickM-27** [reply]
```
record:
  expire_interval:
```

**Comment by jamesonuk** [comment]
I noticed that disks were still spinning up unexpectedly and then I noticed that snapshots (clips) seem to be getting deleted but there is no `expire_interval` for snapshots.   I can't see anything in the full config which suggests there is anyway to control when snapshots are purged?

**Reply by NickM-27** [reply]
There is no way to control this currently. Best way to keep disks from spinning in this manor is to store snapshots on an SSD via separate docker mount to recordings
